modules/qtx_modules_handler.php

```diff
@@ -118,6 +135,7 @@ public static function get_modules_defs() {
                 'plugin'            => true,
                 'incompatible'      => 'qtranslate-slug/qtranslate-slug.php',
                 'manual_activation' => true,
+                'has_settings'      => true,
```

Comment:
Note: I want to generalize the activation feature. Eventually `manual_activation` may disappear.
It's possible that `has_settings` will also evolve to store a callback for the settings to be called directly.

Reply:
I was thinking the same thing... check #1147